### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by gui7545**
here is a French translation of the Courseplay.
Don't worry if you see that the translation is longer than the original, I test everything and nothing is out the hud or on other text
have a nice evening
gui

### CODE REVIEW

File: modDesc.xml [hunk 1]
```diff
@@ -27,582 +27,727 @@
...
 		</text>
 		<text name="CourseWaitpointStart">
 			<en>start</en>
 			<de>weiterfahren</de>
+			<fr>Début</fr>
 		</text>
 		<text name="CourseSave">
 			<en>Save course</en>
 			<de>Kurs speichern</de>
+			<fr>Sauvegarder la course</fr>
```
Comment (JakobTischler) [inline modDesc.xml:195]: Wouldn't "enregistrer" be better?

File: modDesc.xml [hunk 2]
```diff
@@ -27,582 +27,727 @@
...
 		</text>
 		<text name="CPwaitFillLevel">
 			<en>Waiting until fill level is reached</en>
 			<de>Warte bis Füllstand erreicht ist</de>
+			<fr>Attend que le niveau soit atteint</fr>
 		</text>
 		<text name="CPnoCombineInReach">
 			<en>No combine in reach</en>
 			<de>Kein Drescher in Reichweite</de>
+			<fr>Pas de Batteuse en vue</fr>
```
Comment (JakobTischler) [inline modDesc.xml:610]: "en vue" would be "in view", right? Maybe use "à portée de (la) main" ? Although "main" wouldn't be "hand" here, of course

File: modDesc.xml [hunk 3]
```diff
@@ -27,582 +27,727 @@
...
 		</text>
 		<text name="CPWorkingWidht">
 			<en>working width</en>
 			<de>Arbeitsbreite</de>
+			<fr>largeur de travail</fr>
 		</text>
 		<text name="CPWaterDrive">
 			<en>would like to go swimming</en>
 			<de>möchte baden gehen</de>
+			<fr> a voulut prendre un bain</fr>
```
Comment (JakobTischler) [inline modDesc.xml:670]: delete first space before "a"

File: modDesc.xml [hunk 4]
```diff
@@ -27,582 +27,727 @@
...
 		</text>
 		<text name="CrossPoints">
 			<en>crossing points:</en>
 			<de>Kreuzungspunkte:</de>
+			<fr>Points de croisement:</fr>
 		</text>
 		<text name="CPopenHud">
 			<en>open Courseplay:</en>
 			<de>Courseplay öffnen:</de>
+			<fr>Ouverture de Courseplay:</fr>
```
Comment (JakobTischler) [inline modDesc.xml:685]: wouldn't "ouvrir Courseplay" suffice?

**Comment by JakobTischler** [comment]
Please see my added line notes.

**Comment by gui7545** [comment]
"Enregistrer" and "Sauvegarder" have the same meaning: to save something.

"en vue" in my sentence mean: to be in sight and that mean that tractors can't reach to the Combine because there is no Combine or it is too far away.

About "ouvrir" and "ouverture", this text is for choose the action to do to open the Courseplay.
To do this action will bring about the opening of the Courseplay.
that why I thinnk it is "ouverture"

**Comment by JakobTischler** [comment]
Alright, thanks for the explanations. I won't be able to merge it, because the moddesc has since been changed and still will be changed, so I'll add your `<fr>` additions manually.

**Comment by gui7545** [comment]
ok thanks

**Pull request closed without merging**